TestResultSummaryService/parsers/TestBenchmarkParser.js

```diff
@@ -17,31 +16,33 @@ class TestBenchmarkParser extends Parser {
                 benchmarkName = testName.substring(0, index);
                 benchmarkVariant = testName.substring(index+1, testName.length);
             }
+        } 
+        const benchmarkNameKeys = Object.keys(BenchmarkMetric);
+        if(benchmarkName && Array.isArray(benchmarkNameKeys) && benchmarkNameKeys.length > 0){
+            for(let key of benchmarkNameKeys) {
+                if(benchmarkName.toLowerCase().includes(key.toLowerCase())) {
+                    benchmarkNameKey = key;
```

Comment:
At least to me, `benchmarkNameKey` isn't very clear. I personally would prefer to call it something like `benchmarkParserKey` to be more reflective what it holds. Same thing for `benchmarkNameKeys`.

Reply:
I do not think there is a need to have the variable at all. Please see my comment below.

Reply:
Yeah I don't mean that we need a var here! I mentioned it here since this is the origin for the parser key even though that name `benchmarkNameKey` is used in many other places. I'm just suggesting that we should be more specific what that key refers to in our naming.